install.sh

```diff
@@ -1,31 +1,32 @@
 #!/bin/bash
-# For OS X-based systems.
+
+# OS detection
+osdetect=$(uname)
+file_path="unknown"
+if [ "$osdetect" == 'Darwin' ]]; then
+  file_path="/Library/Fonts/"
+elif [[ "$osdetect" == 'Linux' ]]; then
+  if [[ -d /usr/local/share/fonts/ ]]; then # Debian/Ubuntu and others.
+    file_path="/usr/local/share/fonts/"
+  elif [[ -d /usr/share/fonts/ ]]; then # OpenSUSE, Arch and other distros using this directory structure
+    mkdir -p /usr/share/fonts/google/
+    file_path="/usr/local/share/fonts/google/"
+  else # Fallback to installing fonts locally to the user, this is a safe bet, as several distros use this location.
+    mkdir -p ~/.fonts
+    file_path="~/.fonts/"
+  fi
+fi
 
 clear
-echo "Installing all Google Web Fonts onto your Mac"
+echo "Installing all Google Web Fonts onto your System"
 echo "Downloading the fonts..."
-cd ~/Documents/
-curl -L https://github.com/google/fonts/tarball/master -o master.tar.gz
+curl -L https://github.com/google/fonts/tarball/master -o ~/Documents/master.tar.gz
```

Comment:
Using `~/Documents/` seems like a Mac only pattern. This would usually not exist on linux and throw some directory errors. May be better to perform these inside of `/tmp/` as it's a cross platform standard.